src/sid/covid_epi_params.csv

```diff
@@ -125,15 +124,15 @@ cd_received_test_result_true,all,3,0.1
 cd_received_test_result_true,all,4,0.07
 cd_knows_immune_false,all,-1,1
 cd_knows_infectious_false,all,-1,1
-cd_is_immune_by_vaccine,all,-1,0.25
-cd_is_immune_by_vaccine,all,14,0.35
-cd_is_immune_by_vaccine,all,21,0.4
+cd_is_immune_by_vaccine,all,-1,1
```

Comment:
It was used to inform about the number of days since vaccination has passed. I created a new countdown ``cd_ever_vaccinated`` analogously to ``cd_ever_infected`` which is now used to count the number of days since vaccination. All other vaccination related countdowns are deleted.